app/views/content/train-to-be-a-teacher/if-your-application-is-unsuccessful.md

```diff
@@ -0,0 +1,79 @@
+---
+title: If your teacher training application is unsuccessful
+subcategory: Your teacher training application
+description: |-
+  Read about what to do next if your teacher training application is unsuccessful. Find advice on the different reasons an application can be unsuccessful and how to get support.
```

Comment:
slightly too long metadescription (176 characters) - could we cut back to 160 (or under)?